cantera/build.sh

```diff
@@ -11,7 +11,7 @@ echo "boost_inc_dir = '${PREFIX}/include'" >> cantera.conf
 if [[ "${OSX_ARCH}" == "" ]]; then
     echo "CC = '${CC}'" >> cantera.conf
     echo "CXX = '${CXX}'" >> cantera.conf
-    echo "blas_lapack_libs = 'mkl_rt,dl'" >> cantera.conf
+    # echo "blas_lapack_libs = 'mkl_rt,dl'" >> cantera.conf
```

Comment:
Can you add a todo comment here and elsewhere in meta.yaml to note this temporary and that it's commented out to solve #31?